packages/babel-parser/test/plugin-options.js

```diff
@@ -0,0 +1,37 @@
+import Parser from "../lib/parser";
```

Comment:
These tests are failing because the parser is bundled, so only the index is accessible consistently. Looks like these need to be tweaked to test the actual public interface of the parser.